cmd/skaffold/app/cmd/init.go

```diff
@@ -231,14 +231,13 @@ func processBuildArtifacts(pairs []dockerfilePair) latest.BuildConfig {
 	return config
```

Comment:
nit: config -> pipeline

Reply:
I left `BuildConfig` and `DeployConfig` as they are, WDYT?